samples/ApptentiveSample/MainActivity.cs

```diff
@@ -1,49 +1,63 @@
 ﻿using Android.App;
 using Android.Widget;
 using Android.OS;
-using ApptentiveSDK.Android;
+using ApptentiveKit.Android;
 using System;
 using Android.Content;
 using Android.Gms.Common;
-using Firebase.Messaging;
-using Firebase.Iid;
 using Android.Util;
+using Kotlin;
+using Xamarin.Essentials;
+using Android.Support.V7.App;
+using AndroidX.Fragment.App;
+using AndroidX.Lifecycle;
 
 namespace ApptentiveSample
 {
-    [Activity(Label = "ApptentiveSample", MainLauncher = true, Icon = "@mipmap/icon")]
-    public class MainActivity : Activity, IUnreadMessagesListener
```

Comment:
Yes, but legacy doesn't require the Observable method call.

Reply:
Curious if you've tested out the code I added to the slack channel?

Reply:
The code provided doesn't work because in the implementation it still relies on using the observe method which is not being transferred from kotlin to c# in the binding.